src/Integration.Vsix.UnitTests/SonarLintTagger/IssueToFilterableIssueConverterTests.cs

```diff
@@ -0,0 +1,162 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Linq;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.Text;
+using Moq;
+using Sonarlint;
+using SonarLint.VisualStudio.Integration.Suppression;
+using SonarLint.VisualStudio.Integration.Vsix;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests
+{
+    [TestClass]
+    public class IssueToFilterableIssueConverterTests
+    {
+        [DataTestMethod]
+        [DataRow(100, 1)]
+        [DataRow(101, 100)]
+        public void CreateFilterableIssue_IssueLineOutsideSnapshot_ReturnsNull(int issueLine, int bufferLineCount)
```

Comment:
really nitpicking: it took me some time to understand the params as opposed to the their names in the code; maybe rename them to "startLine" and "lineCount"?

Reply:
If the naming is making is harder to understand then it's not nitpicking.
How about `issueStartLine` and `bufferLineCount`? For me, `startLine` and `lineCount` don't convey enough information - I have to read the test to see that they relate to different artefacts.

If something _really_ is just nitpicking then it's probably a waste of everyone's time mentioning it :-)

Reply:
Yep, sounds good 👍